client/src/ducks/authentication/authentication.js

```diff
@@ -51,18 +56,31 @@ export const actions = {
         if (authResult && authResult.accessToken && authResult.idToken) {
           try {
             const decodedIdToken = jwtDecode(authResult.idToken);
+            // TODO - alter API singleton so it auto-syncs to redux state / local storage
```

Comment:
Have you made an issue for this?

Reply:
G00d idea https://github.com/dpopp07/bartop/issues/95

Reply:
👍